frontend/src/custom.d.ts

```diff
@@ -57,6 +57,24 @@ declare type ReviewInput = {
   rating: number;
 };
 
+declare type ProfileSearchResult = {
+  id: number;
+  gitHubId: string;
+  name: string;
+  imageUrl: string;
+  careerLevel: string;
+  jobType: string;
+  profileProducts: [
+    {
+      id: number;
+      name: string;
+      imageUrl: string;
+      reviewCount: number;
+      rating: number;
+      category: string;
+    }
+  ];
+
```

Comment:
```suggestion
  profileProducts: Product[];
```
아마 나중에는 Product 타입을 조금 바꾼 다음에 이런 식으로 바꾸면 좋을 것 같아요!